src/main/java/gyro/google/compute/InstanceNetworkInterface.java

```diff
@@ -150,14 +190,14 @@ public NetworkInterface copyTo() {
             networkInterface.setSubnetwork(getSubnetwork().getSelfLink());
         }
 
-        if (getAccessConfigs() != null) {
-            networkInterface.setAccessConfigs(getAccessConfigs().stream()
+        if (getAccessConfig() != null) {
```

Comment:
If the getter changes are made then
```suggestion
        if (!getAccessConfig().isEmpty()) {
```